articles/language/file-structure.md

```diff
@@ -385,3 +385,28 @@ function DotProduct(a : Double[], b : Double[]) : Double {
     return accum;
 }
 ```
+
+
+## Internal Declarations
+
+User-defined types, operations, and functions can also be declared as *internal*.
+This means that they can only be accessed from within the Q# project that they are declared in.
+When a project is used as a reference, all of its non-internal declarations are made available, but trying to use an internal declaration from another project will produce an error.
+Internal declarations are useful for writing modular code that can be reused by other parts of your project, but still be changed later without breaking other projects that might depend on it.
+
+An internal user-defined type, operation, or function can be declared simply by adding `internal` at the beginning of the declaration.
+For example,
+
+```qsharp
+internal newtype PairOfInts = (Int, Int);
+
+internal operation EPR (q1 : Qubit, q2 : Qubit) : Unit 
```

Comment:
I'd suggest that samples in the docs follow the Q# style guide to help users gain familiarity with the conventions that they'll see when using various Q# APIs. In this case, I'd suggest making the operation name a verb ("prepare" in keeping with Q# API principles), and not using people's names in the operation name.
```suggestion
internal operation PrepareEntangledPair(q1 : Qubit, q2 : Qubit) : Unit 
```

Reply:
I copied this from an earlier example in the page, so I also renamed it there.

Is the convention no space between the name and the argument tuple in the declaration?